packages/block-library/src/blocks/supernova/style.scss

```diff
@@ -10,28 +10,28 @@
 //}
 
 .supernova {
-  //--block-top-spacing: var(--novablocks-block-top-spacing, 0);
-  //--block-bottom-spacing: var(--novablocks-block-bottom-spacing, 0);
-  //--emphasis-top-spacing: var(--novablocks-emphasis-top-spacing, 0);
-  //--emphasis-bottom-spacing: var(--novablocks-emphasis-bottom-spacing, 0);
+  //--block-top-spacing: var(--nb-block-top-spacing, 0);
+  //--block-bottom-spacing: var(--nb-block-bottom-spacing, 0);
+  //--emphasis-top-spacing: var(--nb-emphasis-top-spacing, 0);
+  //--emphasis-bottom-spacing: var(--nb-emphasis-bottom-spacing, 0);
 
   &[class] {
-    --emphasis-area-width: calc( var(--collection-emphasis-area, 100) * 1% );
+    --emphasis-area-width: calc( var(--nb-collection-emphasis-area, 100) * 1% );
```

Comment:
local custom properties like `--emphasis-area-width` can stay unprefixed, but at first glance there are these instances of `--emphasis-are` which seem to be the scope of `--nb-collection-emphasis-area`.
https://github.com/pixelgrade/nova-blocks/pull/358/files#diff-b14d77c67709be570829035f809a29f4f27e2d4089e714a4908bd2e81934157cR92